Scroll-Driven Animations Case Study: Add note about support (#6924)
GoogleChrome/developer.chrome.com: site/en/blog/scroll-animation-performance-case-study/index.md

This article compares the new approach with the classic JavaScript technique to show just how easy and silky-smooth scroll-driven animations can be with the new API.

> {% Aside %} The performance gains that scroll-driven animations give you are available as of Chrome 116. {% endAside %}

Reviewer: ```suggestion
{% Aside %} The following performance gains from the use of scroll-driven animations give you are available from Chrome 116. {% endAside %}
```
